node/config/types.go

```diff
@@ -621,6 +621,9 @@ type IndexConfig struct {
 	// EXPERIMENTAL FEATURE. USE WITH CAUTION
 	// EnableMsgIndex enables indexing of messages on chain.
 	EnableMsgIndex bool
+
+	// EnableAutomaticBackFill enables automatic index back-filling
+	EnableAutomaticBackFill bool
```

Comment:
For now, we need to rename this to `EnableAutomaticBackFillTxIndex` as we're only doing it for the TX Index.

Reply:
Done.